### DISCUSSION TITLE
The /github subscribe [repository name] Command Fails To Subscribe

### DISCUSSION THREAD

**Discussion by arunmalve** [strimzi/strimzi-kafka-operator#8487] [Q&A]
The GitHub app returns the following message when using the `/github subscribe strimzi/strimzi-kafka-operator`

Error Message: 
"Either the app isn't installed on your repository or the repository does not exist. Install it to proceed." despite supplying a valid repo URL and having already installed the GitHub app on the Slack Workspace. 

Please install the Slack app to see the new release notifications.

**Comment by scholzj** [comment]
That is probably a question on your Slack admins. 

If you wanna be informed about new releases, you can also watch the different repositories. To do so, go to the respository (e.g. https://github.com/strimzi/strimzi-kafka-operator), press the "Watch" button in the top right corner nd select what events you want to be notified about. Yu can also join the #strimzi channel on CNCF Slack or our user mailing list to be notified about new releases: https://github.com/strimzi/strimzi-kafka-operator#getting-help

**Comment by arunmalve** [comment]
This is a reference link to the Slack integration in the Repo for notifications to be installed, so we can easily follow any new Releases. [scholzj](https://github.com/scholzj)
https://www.youtube.com/watch?v=pB_nmvjYGQM

thank you so much for your attention and participation.
